app/views/take_whole_pot_calculator/show.html.erb

```diff
@@ -0,0 +1,83 @@
+<% content_for(:data_layer) do %>
```

Comment:
This should be in `_results.html.erb` so that the behaviour applies when the client has JavaScript?